metricbeat/module/kubernetes/event/config.go

```diff
@@ -0,0 +1,35 @@
+package event
+
+import (
+	"errors"
+	"time"
+
+	"github.com/elastic/beats/libbeat/common"
+)
+
+type kubeEventsConfig struct {
+	InCluster  bool          `config:"in_cluster"`
+	KubeConfig string        `config:"kube_config"`
+	Namespace  string        `config:"namespace"`
+	SyncPeriod time.Duration `config:"sync_period"`
+}
+
+type Enabled struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Enabled should have comment or be unexported